docs/examples/feature_embedding.ipynb

```diff
@@ -0,0 +1,432 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[![Open In\n",
+    "Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/alibaba/feathub/blob/master/docs/examples/feature_embedding.ipynb)"
```

Comment:
Can you help make the capability of running the example in CoLab more obvious, e.g. by adding text describing this capability?